examples/pylab_examples/xcorr_demo.py

```diff
@@ -1,8 +1,9 @@
 """
 ==========
-Xcorr Demo
+Cross-correlation Demo
```

Comment:
The title lines on this one need to be the same length as the title (which is what is failing travis)